ci/travis/install-dependencies.sh

```diff
@@ -141,13 +172,20 @@ install_dependencies() {
       pip_packages+=(blist)  # https://github.com/DanielStutzbach/blist/issues/81#issue-391460716
     fi
     CC=gcc pip install "${pip_packages[@]}"
-  elif [ "${LINT-}" = 1 ]; then
+  fi
+
+  if [ "${LINT-}" = 1 ]; then
     install_miniconda
-    pip install flake8==3.7.7 flake8-comprehensions flake8-quotes==2.0.0  # Python linters
-    pushd "${WORKSPACE_DIR}/python/ray/dashboard/client"  # TypeScript & HTML linters
-      install_node
-      run_npm
-    popd
+    pip install flake8==3.7.7 flake8-comprehensions flake8-quotes==2.0.0 yapf==0.23.0  # Python linters
+    # readthedocs has an antiquated build env.
+    # This is a best effort to reproduce it locally to avoid doc build failures and hidden errors.
+    local filename
```

Comment:
why is the below change needed? pip install seems to have worked before just fine

Reply:
The `sed` line I can probably remove (I think I added it a while ago to help me debug some stuff), but the `filter_compatible_pip_requirements` line is needed since the `blist` requirement doesn't work on on Windows as I mentioned below.

Reply:
I think we can remove it from the `requirements-rtd`, because we are mocking it for read the docs anyways. Does that work?

Reply:
Not sure, I can give it a shot and see.

Reply:
Yeah, I'm pretty sure it will work :)